main.js

```diff
@@ -2,3 +2,48 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+
+
+// We loaded axios library in our html file
+// Lets make an ajax call with axios
+
+//create image tag 
+//const img =document.createElement('img') ; 
+//create the button
+const button=document.querySelector('#random') ;
+const container=document.querySelector('.container') ;  
+const apiUrl='https://rickandmortyapi.com/api/character'; 
+button.innerText='Click me ' ; 
+const getImages=() => { //method to call the event for the button
+    
+    axios({
+        url:apiUrl,
+        method:'get',
+      
+        
+    })
+           .then(res  => {
+               console.log(res) ; 
+               for(let i=0;i<5;i++) {
+               const img =document.createElement('img') ; 
```

Comment:
Nice work, 👍
If you read the documantation you can pass an array to the URL and get an array of results.
```
https://rickandmortyapi.com/api/character/[1,2,3,4,5]
```
so instead of calling the API 5 times, you can call it once and get 5 results.